x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/MachineLearning.java

```diff
@@ -1249,60 +1255,137 @@ public String getFeatureDescription() {
         return "Provides anomaly detection and forecasting functionality";
     }
 
-    @Override public void cleanUpFeature(
+    @Override
+    public void cleanUpFeature(
         ClusterService clusterService,
         Client client,
-        ActionListener<ResetFeatureStateResponse.ResetFeatureStateStatus> listener) {
+        ActionListener<ResetFeatureStateResponse.ResetFeatureStateStatus> finalListener) {
+        logger.info("Starting machine learning cleanup");
```

Comment:
```suggestion
        logger.info("Starting machine learning feature reset");
```

(Just to make it crystal clear to support that this was triggered by a user calling the reset API.